app/allocations/middleware/set-results.allocations.js

```diff
@@ -9,12 +9,11 @@ async function setResultsAllocations(req, res, next) {
   }
 
   try {
-    const results = await req.services.allocation.getByDateAndLocation(
+    req.results = await req.services.allocation.getByDateAndLocation(
```

Comment:
Elsewhere we "namespace" these results variables with names like `resultsAsTables` or `resultsAsCards` - is there a more descriptive name we could use for this?

Reply:
The reason for this is that this is a different structure of the object and is used by a different middleware than the other properties.

It's used by the [`set-pagination`](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/ea1555b5e1f0e5e5d9cc5efabd0775f6bf90ae42/common/middleware/collection/set-pagination.js) middleware.

Reply:
Ok, so I guess we have a pattern where we put the raw result on the `req` as `results` and then any modified variants with some `flavour` of namepace, e.g. `resultsAsTables`

Reply:
Essentially, yes. This still just follows the same approach of placing things on the `req` object and then using that in subsequent middleware/controllers.

Reply:
Ok, so I've probably tied myself up in knots avoid using a raw `results` in some of my PMU work then :)

:+1: